common/lib/capa/capa/capa_problem.py

```diff
@@ -62,6 +74,9 @@
 # main class for this module
 
 
+DEFAULT_QUESTION_TEXT = _("You must specify meaningful question text.")
```

Comment:
We can't have links in that will work (at all) with the simple editor. Also, don't put "edX" into the text.